cli-plugins/apps/command-reference/workload_get.hbs.md

```diff
@@ -4,7 +4,7 @@
 
 You can view workload details at whenever. Some details are:
```

Comment:
I'd change
`You can view workload details at whenever. Some details are:`
to
`Some of the workload details in the command output are as follows:`